app/src/main/java/org/p2p/wallet/home/ui/main/empty/epoxy/HomeEmptyController.kt

```diff
@@ -0,0 +1,35 @@
+package org.p2p.wallet.home.ui.main.empty.epoxy
+
+import com.airbnb.epoxy.TypedEpoxyController
+import org.p2p.wallet.home.model.EmptyHomeItem
+import org.p2p.wallet.home.model.HomeBannerItem
+
+class HomeEmptyController(
```

Comment:
Controller in Epoxy collects and renders all the items that are passed into 
like Adapter in classical RecyclerView